### PR TITLE
Added Python code in DFS Section

### PR DESCRIPTION
**Pull request by 1a1242**
Python code wasn't available in DFS Section so added a code solving Topological sort using DFS

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
tabs

**Comment by SansPapyrus683** [comment]
btw can you send AC screen on CSES? just wanna make sure it actually works

**Comment by 1a1242** [comment]
> btw can you send AC screen on CSES? just wanna make sure it actually works

Okay!!!
Course Schedule 
image
Round Trip II
image

File: content/4_Gold/TopoSort.mdx [hunk 1]
```diff
@@ -177,6 +177,62 @@ public class CourseSchedule {
...
+import sys
+
+MAX_N = 10 ** 5
+sys.setrecursionlimit(MAX_N)
+
+
+def dfs(node):
+	for i in graph[node]:
+		if visited[i] == 0:
+			visited[i] = 1
```
Comment (SansPapyrus683) [inline content/4_Gold/TopoSort.mdx:192]: use booleans

File: content/4_Gold/TopoSort.mdx [hunk 2]
```diff
@@ -331,6 +387,63 @@ public class cycle {
...
+			if dfs(u):
+				if on_stack[node] == 1:
+					cycle.append(node)
+					on_stack[node] = 0
+					return True
+				else:
+					cycle.append(node)
+					return False
+			
+			if cycle != []:
```
Comment (SansPapyrus683) [inline content/4_Gold/TopoSort.mdx:417]:
```suggestion
			if not cycle:
```

File: content/4_Gold/TopoSort.mdx [hunk 3]
```diff
@@ -331,6 +387,63 @@ public class cycle {
...
+				return False
+	on_stack[node] = False
+	return False
+
+
+N, M = map(int, input().split())
+adj = [[] for _ in range(N + 1)]
+cycle = []
+visited = [0] * (N + 1)
+on_stack = [0] * (N + 1)
```
Comment (SansPapyrus683) [inline content/4_Gold/TopoSort.mdx:427]: you used 0-indexing in your other solution, why not this one as well?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by SansPapyrus683 — APPROVED** [review]

File: content/4_Gold/TopoSort.mdx [hunk 4]
```diff
@@ -331,6 +387,63 @@ public class cycle {
...
+			if not cycle:
+				return False
+	on_stack[node] = False
+	return False
+
+
+N, M = map(int, input().split())
+adj = [[] for _ in range(N)]
+cycle = []
+visited = [False] * )
```
Comment (ryanchou-dev) [inline content/4_Gold/TopoSort.mdx:425]: what

**Review by ryanchou-dev — APPROVED** [review]
yayy

**Comment by 1a1242** [comment]
@andrewwangva @dongliu0426 seem inactive.
Is there anyway that I can contact them?
Or this pull request might get merged?

**Pull request merged**